src/lib/index.js

```diff
@@ -66,6 +66,18 @@ define("WherebyEmbed", {
     },
 
     // Commands
+    startRecording() {
+        if (this.iframe.current) {
+            const url = new URL(this.room, `https://${this.subdomain}.whereby.com`);
```

Comment:
Would be nice to encapsulate this logic in a helper function so we don't have to repeat it in every command. DRY 🤓 (first time a quote such a thing)

Reply:
Agree, will fix

Reply:
Fixed